app/src/main/java/de/storchp/opentracks/osmplugin/maps/CompassManager.java

```diff
@@ -8,7 +8,7 @@
 import android.view.animation.RotateAnimation;
 import android.widget.ImageView;
 
-public class Compass  implements SensorEventListener {
+public class CompassManager implements SensorEventListener {
```

Comment:
I personally don't like the *Manager postfix, but I understand that you want to distinguish between the Class with the Sensors and the Image.

Reply:
Totally agree - I am not that happy myself.
What about CompassHandler or CompassListener or CompassCallback?

Reply:
Handler feels the same as Manager, if you really want to rename it then CompassListener.
I'm thinking of a self contained widget named Compass. But can be done in a later refactoring.

Reply:
Done ;)